frontend/src/views/Comment.svelte

```diff
@@ -0,0 +1,86 @@
+<style>
+  .comment {
+    white-space: pre;
+  }
+
+  button {
+    border: 0px;
+    filter: invert(20%);
+  }
+
+  button:hover {
+    filter: invert(40%);
```

Comment:
I wonder if it makes sense to use one of the animal color CSS variables here? Any particular reason for using the invert filter?